python/src/scipp/plotting/figure3d.py

```diff
@@ -391,31 +411,32 @@ def update_colorbar(self):
                          cmap=self.scalar_map.get_cmap(),
                          norm=self.scalar_map.norm,
                          extend=self.extend)
-        cbar_ax.set_ylabel(self.unit)
+        # TODO Why does this not show up?
+        cbar_ax.set_ylabel(self._unit)
```

Comment:
@nvaytet Trying to figure out why unit is not visible. The label is actually there, but it is either cut off when converting to image or hidden otherwise. It does appear if I change the label coords, but I can never get it to display whitespace for it between viewport and colorbar. Any clue why this isn't working anymore?

Reply:
so you mean it is being clipped when making the image for the colorbar?

Reply:
Yes.

Reply:
hmm maybe try changing https://github.com/scipp/scipp/blob/18c1ff2cfcba50230811f2d6a406195be8eafeaf/python/src/scipp/plotting/tools.py#L194
to
```Py
fig.savefig(buf, format='png', bbox_inches='tight')
```

Reply:
No effect.

Reply:
It kind of works with that if I don't set the label position. It then shows up on the *right* instead of left, but better than nothing...